### DISCUSSION TITLE
Color coding of protein model

### DISCUSSION THREAD

**Discussion by kmradford** [hodcroftlab/covariants#208] [Q&A]
Hi there! 

CoVariants is amazing! Thank you for creating and maintaining this fantastic resource. Our lab is curious about the chemical changes on the surface of the Delta variant Spike, and it's really cool to see them mapped out on the protein model. However, we're not sure what the color coding means, and I haven't found a key on the site anywhere. Is there a key somewhere that you could refer me to?

Thank you so much! 
Best,
Kate

**Comment by emmahodcroft** [comment]
Dear Kate,
Thanks so much for the kind words!

Unfortunately I don't know much about the colour-coding on the protein structures myself. I generate these from https://corona.bii.a-star.edu.sg/ , and it automatically adds the colours - for example for Delta: 
![image](https://user-images.githubusercontent.com/14290674/129549648-ad190de9-ebcb-404e-aa9f-4265a5cf0ac1.png)
However, from clicking on these changes, I can't really make out what the colours mean exactly. T19R does have a text in the same colour which says the mutation "removes a potential N-glycosylation site", but none of the others have text in 'matching colours' which could underscore that the colour means something. The colours don't seem to correspond well to the 'areas' of Spike (RDB, etc). So I'm afraid I'm at a loss myself!

I'd love to incorporate a better, possibly interactive protein viewer, but so far it hasn't made it up to the top of the to-do list 🙃 But hopefully sometime!

Sorry I couldn't be of more help!
Best wishes,
Emma

**Reply by CarlijnB** [reply]
Hi @kmradford and @emmahodcroft, 

I'm not an expert on Covsurver, but I've come across it in my research as well.
Here's the colour coding explained:   [edited with info for covsurver , rather than flusurver]

https://corona.bii.a-star.edu.sg/faq.html#Question8
The aa changes are color-coded according to their known or predicted biological effect and epidemiological significance. When there are no known effects for the aa change and the aa change occurred only once in the current set of sequences, the aa change will appear in black colored font. AA changes occurring more than 100 times are more interesting epidemiologically and will appear in blue colored font. If the aa change occurs at a site known to be involved in phenotypic effects such as altering host-cell receptor binding or antigenicity, it will appear in orange if the site is shared but not the type of amino acids involved. AA changes that create or remove a potential glycosylation site are colored magenta, and aa changes that lead to an insertion or deletion of amino acid residues are colored in cyan. Only if the phenotype change reported before matches the same amino acid types seen in the query we color green for predicted neutral effect and red for predicted enhancing effect. It is crucial to read the associated literature (link provided) to make final judgement on the expected effect of any aa change.


Hope this helps! 

Carlijn

**Reply by emmahodcroft** [reply]
Thanks @CarlijnB that is super helpful!

**Comment by kmradford** [comment]
Hi Emma and Carlijn,

Thank you so much for getting back to me! This is super helpful, and exactly the information I needed! Woo!

Best wishes,
Kate


Kate Radford (she/her/hers)
Graduate student (G1), Biochemistry and Molecular Biophysics
​
California Institute of Technology​​
1200 E California Blvd
Pasadena, CA 91125
________________________________
